v21.2/ui-sessions-page.md

```diff
@@ -6,11 +6,13 @@ docs_area: reference.db_console
 ---
 
 {{site.data.alerts.callout_info}}
-On a secure cluster, this area of the DB Console can only be accessed by a SQL user with the `VIEWACTIVITY` role option. Note that non-`admin` users will see only their own sessions, while `admin` users see sessions for all users.
+On a secure cluster, this area of the DB Console can only be accessed by a SQL user with the `VIEWACTIVITY` [role option](alter-role.html#role-options). Non-`admin` users will see only their own sessions, while `admin` users see sessions for all users.
 {{site.data.alerts.end}}
 
-The **Sessions** page provides details of all open sessions in the cluster.
+The **Sessions** page  of the DB Console provides details of all open sessions in the cluster.
```

Comment:
this page exists on both DB and CC Console, so I don't think we should be saying only DB Console here (do we even use this term publicly?)

Reply:
Yes, we do use the term DB Console publicly: https://deploy-preview-13623--cockroachdb-docs.netlify.app/docs/dev/ui-overview.html